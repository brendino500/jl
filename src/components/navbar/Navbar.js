import { makeStyles } from '@material-ui/core/styles'
import { AppBar, Toolbar } from '@material-ui/core/'

import UserButtons from './UserButtons'
import logo from '../../assets/se-logo-white.png'
import whiteBackground from '../../assets/user-login-background.png'

const useStyles = makeStyles((theme) => ({
  root: {
    flexGrow: 1,
    color: '#000000',
    margin: 0,
  },
  navbar: {
    backgroundColor: '#000000',
    overflow: 'hidden',
  },
  menuButton: {
    marginRight: theme.spacing(2),
  },
  title: {
    flexGrow: 1,
  },

  buttonSection: {
    position: 'absolute',
    right: 0,
  },
  userButtons: {
    position: 'absolute',
    top: '1.2em',
    right: '2em',
  },
  loginBackground: {
    width: '17em',
    height: '4.5em',
  },
}))
export default function Navbar() {
  const classes = useStyles()

  return (
    <div className={classes.root}>
      <AppBar
        position="fixed"
        className={classes.navbar}
        style={{
          boxShadow: '0px 0px 0px 0px',
        }}
      >
        <Toolbar>
          <img src={logo} alt="logo" />

          <div className={classes.buttonSection}>
            <img
              src={whiteBackground}
              alt="login background"
              className={classes.loginBackground}
            />
            <div className={classes.userButtons}>
              <UserButtons text="Join" />
              <UserButtons text="Login" />
            </div>
          </div>
        </Toolbar>
      </AppBar>
    </div>
  )
}
