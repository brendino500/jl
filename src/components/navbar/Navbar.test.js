import renderer from 'react-test-renderer'
import Navbar from './Navbar'

describe('navbar snapshot test', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Navbar />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
