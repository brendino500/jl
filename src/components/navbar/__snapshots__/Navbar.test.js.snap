// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`navbar snapshot test renders correctly 1`] = `
<div
  className="makeStyles-root-1"
>
  <header
    className="MuiPaper-root MuiAppBar-root MuiAppBar-positionStatic MuiAppBar-colorPrimary makeStyles-navbar-2 MuiPaper-elevation4"
  >
    <div
      className="MuiToolbar-root MuiToolbar-regular MuiToolbar-gutters"
    >
      <img
        alt="logo"
        src="se-logo-white.png"
      />
      <div
        className="makeStyles-buttonSection-5"
      >
        <img
          alt="login background"
          className="makeStyles-loginBackground-7"
          src="user-login-background.png"
        />
        <div
          className="makeStyles-userButtons-6"
        >
          <a
            aria-disabled={false}
            className="MuiButtonBase-root MuiButton-root MuiButton-text makeStyles-buttonText-8"
            href="http://square-enix-games.com"
            onBlur={[Function]}
            onDragLeave={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            rel="noreferrer"
            tabIndex={0}
            target="_blank"
          >
            <span
              className="MuiButton-label"
            >
              Join
            </span>
          </a>
          <a
            aria-disabled={false}
            className="MuiButtonBase-root MuiButton-root MuiButton-text makeStyles-buttonText-8"
            href="http://square-enix-games.com"
            onBlur={[Function]}
            onDragLeave={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            rel="noreferrer"
            tabIndex={0}
            target="_blank"
          >
            <span
              className="MuiButton-label"
            >
              Login
            </span>
          </a>
        </div>
      </div>
    </div>
  </header>
</div>
`;
